0.download_data/scripts/nbconverted/download_data.py

```diff
@@ -9,29 +9,51 @@
 import pandas as pd
 import pathlib
 
+import sys
+sys.path.append("../utils")
+from download_utils import parse_outline_data, combine_datasets
```

Comment:
Consider removing `parse_outline_data`, which doesn't appear to be used within this file.

```suggestion
from download_utils import combine_datasets
```